app/views/state_file/questions/nj_household_rent_own/edit.html.erb

```diff
@@ -5,22 +5,49 @@
 
 <% content_for :card do %>
   <h1 class="h2"><%= title %></h1>
-  <p><%= t(".label", filing_year: current_tax_year) %></p>
+  <p><%= t(".description_html") %></p>
   <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
     <div class="white-group">
       <%= f.cfa_radio_set(
             :household_rent_own,
-            label_text: t(".label"),
+            label_text: t(".label_html", filing_year: current_tax_year),
             collection: [
-              { value: :own, label: t(".homeowner") },
-              { value: :rent, label: t(".tenant") },
-              { value: :both, label: t(".both") },
-              { value: :neither, label: t(".neither") },
-            ],
-            legend_class: "sr-only"
```

Comment:
[dust] what are the reasons/implications around removing `legend_class: "sr-only"`? from the screenshots it looks fine to me so I don't think it blocks, but just wondering why.